spec/models/account_spec.rb

```diff
@@ -381,6 +381,12 @@
       expect(account_2).to model_have_error_on_field(:username)
     end
 
+    it 'is invalid if the username is reserved' do
+      account = Fabricate.build(:account, username: 'support')
+      account.valid?
```

Comment:
Maybe `expect(account).not_to be_valid?` here? - to both trigger calling of valid to populate errors, but also to assert that the the `valid?` call result is false?